### PR TITLE
Mat1356 measure bundle used resources

### PR DESCRIPTION
**Pull request by danmee10**
Requires https://github.com/MeasureAuthoringTool/fhir-mongoid-models/pull/21

Pull requests into cqm-parsers require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [ ] This pull request describes why these changes were made.
- [ ] Internal ticket for this PR: https://jira.cms.gov/browse/MAT-1356
- [ ] Internal ticket links to this PR
- [ ] Code diff has been done and been reviewed
- [ ] Tests are included and test edge cases
- [ ] Tests have been run locally and pass

**Reviewer 1:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

**Reviewer 2:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

### CODE REVIEW

File: test/unit/measure-loader/bundle_loader_test.rb [hunk 1]
```diff
@@ -22,8 +22,12 @@ def test_extract_measure
       assert_equal 'CMS104', measure.fhir_measure.title.value
       assert_equal '42BF391F-38A3-4C0F-9ECE-DCD47E9609D9', measure.set_id, 'Measure set Id does not match expected value.'
       assert_equal 5, measure.libraries.size, 'Mismatching library size.'
-      assert_equal 46, measure.value_set_ids.count, 'Mismatching number of value set Ids.'
+      # Not sure whether this association was a hmbt at one point or if this was never passing, but
+      # value_set_ids doesn't come with the embeds_many :value_sets relation.
+      # assert_equal 46, measure.value_set_ids.count, 'Mismatching number of value set Ids.'
```
Comment (adongare) [inline test/unit/measure-loader/bundle_loader_test.rb:27]: right, this is changed. ValueSets are embedded now. We will need a task to clean that up
